src/scripts/domain/moveCalculator.ts

```
@@ -59,9 +115,9 @@ export class MoveCalculator {
     const moves: Move[] = [];
 
     if (piece.isPromoted) {
-      // TBD
+      this.addKingMoves(board, moves, piece);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)